Misc/NEWS.d/next/Library/2025-02-01-14-55-33.gh-issue-129559.hQCeAz.rst

```diff
@@ -0,0 +1,3 @@
+Add :meth:`bytearray.resize` method to :class:`bytearray` wrapping
+:c:func:`PyByteArray_Resize` so :class:`bytearray` can be efficiently
+resized in place.
```

Comment:
I don't think that PyByteArray_Resize() implementation detail should be mentioned here:

```suggestion
Add :meth:`bytearray.resize` method so :class:`bytearray` can be efficiently
resized in place.
```